src/Blazored.Toast.TestExtensions/InMemoryToast.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Blazored.Toast.Services;
+using Bunit;
+using Microsoft.AspNetCore.Components;
+
+namespace Blazored.Toast.TestExtensions
+{
+    public class InMemoryToast
+    {
+        private readonly ToastLevel _toastLevel;
+        private readonly RenderFragment _message;
+        private readonly string _heading;
+
+        public InMemoryToast(ToastLevel toastLevel, RenderFragment message, string heading)
+        {
+            _toastLevel = toastLevel;
+            _message = message;
+            _heading = heading;
+        }
+
+        /// <summary>
+        /// Checks toast element is rendered with level.
+        /// </summary>
+        /// <param name="level">The toast level expected.</param>
+        public bool HasLevel(ToastLevel level)
+            => _toastLevel == level;
+
+        /// <summary>
+        /// Checks toast element is rendered with message.
+        /// </summary>
+        /// <param name="message">The toast message expected.</param>
+        public bool HasMessage(string message)
+        {
+            using var ctx = new TestContext();
+            var cut = ctx.Render(_message);
+
+            cut.MarkupMatches(message);
+            return true;
+        }
```

Comment:
This can give problems if the `_message` render fragment contains components that needs special configuration of the TestContext to render. Then users will have their test break because they call `HasMessage`, which they dont expect.

Reply:
Thanks!
I have removed this method, allowing users to test as they see fit,